index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Ask It</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="styles/common.css">
+		<link rel="stylesheet" href="styles/header/header.css">
+		<link rel="stylesheet" href="styles/landing/landing.css">
+		<link rel="stylesheet" href="styles/guide/guide.css">
+	</head>
+	<body>
+		<header class="header">
+			<div class="header__logo-container">
+				<img class="header__logo-image" src="shared/header-logo.png">
+			</div>
+			<div class="header__navigation-container">
+				<nav class="header__navigation-list">
+					<div class="header__navigation-item" id="home">
+						<a href="#home" class="header__navigation-link">Home</a>
+					</div>
+					<div class="header__navigation-item" id="howitworks">
+						<a href="#howitworks" class="header__navigation-link">How it works</a>
+					</div>
+					<div class="header__navigation-item" id="features">
+						<a href="#features" class="header__navigation-link">Features</a>
+					</div>
+					<div class="header__navigation-item" id="partners">
+						<a href="#partners" class="header__navigation-link">Partners</a>
+					</div>
+					<div class="header__navigation-item" id="events">
+						<a href="#events" class="header__navigation-link">Events</a>
+					</div>
+					<div class="header__navigation-item" id="about">
+						<a href="#about" class="header__navigation-link">About us</a>
+					</div>
+				</nav>
+			</div>
+			<div class="header__auth-container">
+				<a href="#" class="header__auth-item">
+						<img class="header__auth-icon" src="shared/sign-in-icon.png"></span>
+						<span class="header__auth-item_small-text">Log in</span>
+				</a>
+				<a href="#" class="header__auth-item">
+					<img class="header__auth-icon" src="shared/avatar-icon.png"></span>
```

Comment:
please provide `alt = ...` attribute for image, so that user will see some text in case the image wasn't loaded